wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -0,0 +1,791 @@
+
+Bad example:
+
+```markdown
+HTML comments <!-- TODO explain HTML comments --> should be used for marking to-dos or annotate text.
+```
+
+Good example:
+
+```markdown
+HTML comments<!-- TODO explain HTML comments --> should be used for marking to-dos or annotate text.
+```
+
+## Editing
+
+### End of line sequence
+
+*Caution: Uploading Markdown files using `CRLF` (carriage return and line feed) via GitHub will result in those files using `CRLF`. To prevent this, set the line ending to `LF` (line feed) before uploading.*
+
+Markdown files must be checked in using the `LF` end of line sequence.
+
+### Escaping
+
+*Caution: Titles are parsed as plain text and must not be escaped.*
+
+Escaping Markdown syntax should be used when needed.
+
+### Paragraphs
+
+Each paragraph must be followed by one empty line.
+
+### Line breaks
+
+Line breaks must use a backslash (`\`).
+
+Line breaks must be used sparingly.
+
+## Hatnote
+
+*Not to be confused with [Notice](#notice).*
+
+Hatnotes are short notes placed at the top of an article or section to help readers navigate to related articles or inform them about related topics.
+
+Hatnotes must be italicised and be placed immediately after the heading. If multiple hatnotes are used, they must be on the same paragraph separated with a line break.
+
+### Main page
+
+*Main page* hatnotes direct a reader to the main article of the topic. When this hatnote is used, it implies that the section it is on is a summary of what the linked page is about. This hatnote should have only one link, but may use more if needed. These must be formatted as follows:
+
+```markdown
+*Main page: {article}*
+
+*Main pages: {article} and {article}*
+```
+
+### See also
+
+*See also* hatnotes suggest to readers other points of interest from a given article or section. These must be formatted as follows:
+
+```markdown
+*See also: {article}*
+
+*See also: {article} and {article}*
+```
+
+### For see
+
+*For see* hatnotes are similar to *see also* hatnotes, but are generally more descriptive and direct. This hatnote should have only one link, but may use more if needed. These must be formatted as follows:
+
+```markdown
+*For {description}, see: {article}`*
+
+*For {description}, see: {article} and {article}`*
+```
+
+### Not to be confused with
+
+*Not to be confused with* hatnotes help distinguish ambiguous or misunderstood article titles or sections. This hatnote should have only one link, but may use more if needed. These must be formatted as follows:
+
+```markdown
+*Not to be confused with {article}.*
+
+*Not to be confused with {article} or {article}.*
+```
+
+### For other uses
+
+*For other uses* hatnotes are similar to *not to be confused with* hatnotes, but links directly to the disambiguation article. This hatnote must only link to the disambiguation article. These must be formatted as follows:
```

Comment:
think it'd be a good rule to put one of these at the top of every article a specific disambiguation page links to

Reply:
Alright, I'm thinking to add this to the **Disambiguation articles** section:
```
Articles linked from a disambiguation article must have a [For other uses](#for-other-uses) hatnote.
```

Reply:
that is most definitely it